ui/src/components/DataTable.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { DataGrid ,ColDef , RowData} from '@material-ui/data-grid';
+import axios, {AxiosRequestConfig, AxiosResponse} from "axios";
+
+interface ArrivalsData {
+  data: RowData[];
+}
+
+interface IProps {
+  region: string;
+  country: string;
+  date: string;
+}
+
+interface IState {
+  arrivalRows?: RowData[]
+  arrivalsData?: ArrivalsData
```

Comment:
Looks like `arrivalsData` isn't used so can be removed from here and `updateFlightsData`